examples/sudoku.jl

```diff
@@ -60,20 +61,25 @@ function SolveModel(initgrid)
     end
 
     # Solve it
-    status = solve(m)
+    JuMP.optimize!(m)
+
+    status = JuMP.termination_status(m)
+    primal_status = JuMP.primal_status(m)
+
+
 
     # Check solution
-    if status == :Infeasible
-        error("No solution found!")
-    else
-        mipSol = getvalue(x)
+    if status == MOI.Success && primal_status == MOI.FeasiblePoint
+        mipSol = JuMP.result_value.(x)
         sol = zeros(Int,9,9)
         for row in 1:9, col in 1:9, val in 1:9
             if mipSol[row, col, val] >= 0.9
                 sol[row, col] = val
             end
         end
         return sol
+    else
+        error("No solution found!")
```

Comment:
No solution found is misleading. `primal_status == MOI.FeasiblePoint` means that the solver has a feasible solution, regardless of `status`. (It may have found a feasible solution before proving optimality.)

Reply:
This wasn't my wording :)   I just translated old existing examples for the new grammar. 

For this particular problem, can we say "No optimal solution found"?

Reply:
> This wasn't my wording :) I just translated old existing examples for the new grammar.

This wasn't meant to suggest that you did something wrong, just that we need to update this code to be correct. People will be looking at these examples to understand how statuses work.

> For this particular problem, can we say "No optimal solution found"?

I'd prefer "The solver did not find an optimal solution." to be a bit more explicit.